package.json

```
@@ -26,7 +26,7 @@
     "graphql-js"
   ],
   "engines": {
-    "node": "^16.19.0 || ^18.14.0 || >=19.7.0"
+    "node": ">=20.0.0"
```

Let's set the required version to those that support `require(esm)` even if we don't end up needing it ourselves, we might need it to update our dependencies.

```suggestion
    "node": "^20.19.0 || ^22.12.0 || >=24.0.0"
```

---

We should at least exclude the EOL releases 21.x and 23.x - we don't want to support these.

```suggestion
    "node": "^20.0.0 || ^22.0.0 || >=24.0.0"
```